koku/masu/api/running_celery_tasks.py

```diff
@@ -44,3 +45,14 @@ def celery_queue_lengths(request):
     queue_len = collect_queue_metrics()
     LOG.info(f"Celery queue backlog info: {queue_len}")
     return Response(queue_len)
+
+
+@never_cache
+@api_view(http_method_names=["GET"])
+@permission_classes((AllowAny,))
+@renderer_classes(tuple(api_settings.DEFAULT_RENDERER_CLASSES))
+def clear_celery_queue(request):
+    """Clear the celery queue."""
```

Comment:
We may want to add a second option to this with a separate param in case it fails to clear the task:
```
import redis
from django.conf import settings
r = redis.Redis(settings.REDIS_URL)
r.flushall()
```

So, that way we can try to clear the tasks using celery first, but if that doesn't work we can quickly give the redis option a try as well.

Reply:
👍🏿  thanks for the suggestion @myersCody